CMakeModules/aseba_conf.cmake

```diff
@@ -0,0 +1,10 @@
+add_library(aseba_conf INTERFACE)
```

Comment:
I do not understand what is this library, could you please add documentation here?

Reply:
It's a "virtual library", if you want. It has no source.
It however sets a number of properties that are used by its dependencies ( transitively )

In particular, it sets a few defines ( ASEBA_ASSERT, WIN32_LEAN_AND_MEAN ), include paths (src), as well as features and properties.

It's equivalent of setting global properties, albeit cleaner since you can write a target that do not "link" on aseba_conf if you want for some reason not to inherit its properties.

Makes sense ?

https://cmake.org/cmake/help/v3.10/manual/cmake-buildsystem.7.html#build-specification-and-usage-requirements

Reply:
> Makes sense ?

Yes, thanks for the explanation.